src/dotnet/Program.cs

```diff
@@ -114,6 +114,8 @@ internal static int ProcessArgs(string[] args, ITelemetry telemetryClient)
                 }
                 else
                 {
+                    firstTimeUseSentinelPath = ConfigureDotNetForFirstTimeUse();
```

Comment:
Just to call it out - we set "DOTNET_SKIP_FIRST_TIME_EXPERIENCE=1" in our CI builds.  Thus we won't get any telemetry in them.

---
In reply to: [67251047](https://github.com/dotnet/cli/pull/3579#discussion_r67251047) [](ancestors = 67251047)